development/sass/top-nav.scss

```diff
@@ -0,0 +1,72 @@
+.navbar {
+  background-color: #2d2d30;
+  font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
+  height: 50px;
+  letter-spacing: 2px;
+  line-height: 20px;
+  opacity: 0.9;
+  text-transform: uppercase;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+
+  button {
+    display: none;
+    padding: 3px 7px;
+    margin: 6px 5px;
+    background-color: #2d2d30;
+    border-radius: 4px;
+  }
+
+  button:focus {
```

Comment:
It's actually more standard to nest this inside the button class you wrote directly above like so: 
```
button {
    display: none;
    padding: 3px 7px;
    margin: 6px 5px;
    background-color: #2d2d30;
    border-radius: 4px;

    &:focus {
      outline: none;
      background-color: #ddd;
    }
}
```
Notice when referring to yourself inside a class you use the ampersand symbol &:hover, &:active, etc.